packages/next-templates/src/app/meldingen/page.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+import { Article, Page, PageContent } from '@utrecht/component-library-react';
+import { TopTaskNavigation } from '@/components/TopTask/TopTaskNavigation';
+import { TopTaskLink } from '@/components/TopTask/TopTaskLink';
+import { ExampleHeader } from '@/components/ExampleHeader/ExampleHeader';
+import { ExampleFooter } from '@/components/ExampleFooter/ExampleFooter';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+export default function Home() {
+  return (
+    <Page>
+      <ExampleHeader />
+      <ExampleNavigation />
+      <PageContent className="voorbeeld-page-content-flex">
```

Comment:
betere class name voor dit is voorbeeld-page-content_flex

Reply:
Goed gespot @scar055! Ik zou er zelfs voorbeeld-page-content--flex doen. (block--modifier) in plaats van block__element)